### PR TITLE
Update index.d.ts to allow custom parameters in RefreshTokenParams interface

### PR DESCRIPTION
**Pull request by Vishwa52**
This change allows to add custom parameters in refresh token request. I used it for adding custom parameter at action level for adding custom claims to the access token.

### CODE REVIEW

File: types/react-native-auth0/index.d.ts [hunk 1]
```diff
@@ -74,6 +74,7 @@ export interface PasswordRealmParams {
 export interface RefreshTokenParams {
     refreshToken: string;
     scope?: string;
+    [key: string]: string;
```
Comment (ascariandrea) [inline types/react-native-auth0/index.d.ts:77]: I think is better to add the `?` to make it optional

**Review by ascariandrea — CHANGES_REQUESTED** [review]

File: types/react-native-auth0/index.d.ts (hunk 1 above)
Comment (Vishwa52) [reply types/react-native-auth0/index.d.ts:77]: key value string pair is optional by default no need to make it optional explicitly.

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## react-native-auth0 ([<kbd>unpkg</kbd>](https://unpkg.com/browse/react-native-auth0@latest/))
was missing the following properties:
1. TimeoutError
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 243720b6934fd1873df4bbb838875c8e573e81b5
</p>

**Review by ascariandrea — APPROVED** [review]

**Review by ascariandrea — APPROVED** [review]

**Comment by Vishwa52** [comment]
Ready to merge

**Pull request merged**